layers/+lang/rust/funcs.el

```diff
@@ -82,24 +82,33 @@ If `help-window-select' is non-nil, also select the help window."
 
 ;; Misc
 
+(defvar spacemacs/rust-quick-run-tmp-file nil
+  "Stores filename for the rust-quick-run function")
+
+(defun spacemacs//rust-quick-run-generate-tmp-file-name (input-file-name)
+  (concat temporary-file-directory
+          (file-name-nondirectory (buffer-file-name))
+          "-"
+          (md5 (buffer-file-name))))
+
+(defun spacemacs//rust-quick-run-compilation-finish-function (buffer status)
+  (if (and (with-current-buffer buffer (string-match "rustc -o /tmp" (buffer-string)))
```

Comment:
i guess the `"/tmp"` part isn't going to work, since it is being populated by `temporary-file-directory` :(